src/QMCWaveFunctions/tests/test_RotatedSPOs.cpp

```diff
@@ -395,12 +397,34 @@ TEST_CASE("RotatedSPOs exponentiate matrix", "[wavefunction]")
 
   CheckMatrixResult check_matrix_result3 = checkMatrix(m3, expected_m3, true);
   CHECKED_ELSE(check_matrix_result3.result) { FAIL(check_matrix_result3.result_message); }
+#ifdef QMC_COMPLEX
+  //Going to test exponentiating a complex antihermitian matrix.
+  using cmplx_t                           = std::complex<RealType>;
+  std::vector<cmplx_t> m3_input_data_cplx = {cmplx_t(0, 0),       cmplx_t(0.3, 0.1),   cmplx_t(0.1, -0.3),
```

Comment:
It would be nice if this example (and the log) were added to `gen_matrix_ops.py`.  However, that script needs some changes to make it easier to add new examples, so maybe not for this PR.

Reply:
I actually hacked gen_matrix_ops.py to generate these reference values, but it was kinda nasty and ad hoc and I was ashamed to add it.

Reply:
That happens.  With the reorganized code in #4818, making a cleaner version should be easier.    Creating a new function `create3x3_complex_matrix` that uses numpy (instead of Sympy) should work.  (And some of output code may need updating, depending on whether you want to just print the values vs.  outputting them in a format ready to use in the unit tests)

Reply:
Oh that's a LOT better.  I can either add it after #4818 is merged prior to this merge, or I can do a follow up PR after they are both in.

Reply:
If there's no urgency to get this PR in, wait for #4818, and then update this PR.

Reply:
Added reference value generation to gen_matrix_ops.py.